Tools/RemoteConsole/ly_remote_console/tests/unit/test_remote_console_commands.py

```diff
@@ -141,11 +141,11 @@ def test_HandleMessage_AutoCompleteList_NoOp(self):
 
     @mock.patch('socket.socket', mock.MagicMock())
     @mock.patch('ly_remote_console.remote_console_commands.threading', mock.MagicMock())
-    def test_HandleMessage_AutoCompleteListDone_ReadySet(self):
+    def test_HandleMessage_ConnectMessage_ReadySet(self):
         rc_instance = remote_console.RemoteConsole()
         rc_instance.on_display = mock.MagicMock()
         rc_instance.ready = mock.MagicMock()
-        msg = b'70'  # in python3 socket.recv returns byte array. 7 is AUTOCOMPLETELISTDONE
```

Comment:
Does this comment need to be updated since the number was updated?

Reply:
Updated comment